docs/guide/start-installation.md

```diff
@@ -37,7 +37,7 @@ by running `composer self-update`.
 
 With Composer installed, you can install Yii by running the following commands under a Web-accessible folder:
 
-    composer global require "fxp/composer-asset-plugin:~1.0.0"
+    composer global require "fxp/composer-asset-plugin:~1.0.3"
```

Comment:
`~1.0.0` works fine i.e. it will install 1.0.3 and 1.0.4 when it will be released.